src/coreclr/interpreter/compiler.cpp

```diff
@@ -825,16 +849,29 @@ void InterpCompiler::EmitCode()
     }
 
     int32_t *ip = m_pMethodCode;
+    bool insideFunclet = false;
     for (InterpBasicBlock *bb = m_pEntryBB; bb != NULL; bb = bb->pNextBB)
     {
+        bool enteringFunclet = false;
```

Comment:
Seems weird that there is quite a bit of logic around computing `enteringFunclet` and passing it over, and it is only used for one assertion.